Gemfile

```diff
@@ -12,6 +12,6 @@ group :test do
   gem 'rubocop', '~> 0.49.0'
 
   platform :mri do
-    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'bd53db95de3d44d631e7c5a267c3d934e66107dd'
+    gem 'liquid-c', github: 'Shopify/liquid-c', branch: 'fix-rtrim-followed-by-tag'
```

Comment:
I will have to wait until https://github.com/Shopify/liquid-c/pull/39 is merged before updating back to the latest commit on liquid-c's master branch